Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:8.1-apache
+FROM php:8.2-apache
```

Comment:
This `Dockerfile` does not attempt to run a `composer install`, and is therefore pretty much useless :scream: